jdaviz/core/loaders/importers/importer.py

```diff
@@ -143,7 +149,10 @@ def add_to_data_collection(self, data, data_label=None, show_in_viewer=True):
         if data_label is None:
             data_label = self.data_label_value
         if hasattr(data, 'meta'):
-            data.meta = standardize_metadata(data.meta)
+            try:
+                data.meta = standardize_metadata(data.meta)
+            except TypeError:
+                pass
```

Comment:
Any chance we would care about this TypeError for non-lcviz configs?

Reply:
ummm, I'm not sure, if this does fail, then the metadata remains as it was, so I think that's ok to silently skip (although could be hard to debug in some cases).  Alternatively we could hardcode cases that should go through `standardize_metadata` so that errors are raised if something goes wrong.  Not sure which is better - what do you think?

Reply:
Gotcha, I think it's fine to leave as-is then. I was thinking of adding an if case to check if it's LCViz for pass vs raise, but if nothing catastrophic happens I think it's ok.